Codecademy/gamut
# feat(GlobalHeader): Mobile Resources menu

### Overview


We need responsive styling for mobile and tablet screens for the new Resources dropdown.


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/40NSFXquATvE19Gm9pnJzI/Catalog-navigation-dropdown?node-id=922%3A13998
- [x] Related to JIRA ticket: DISC-688

#### Testing Instructions



On both portal and monolith PR links below, open the resources menu in both mobile and tablet size screens and make sure it looks like design (except for columns in tablet view). Approved by Tamar ✅
#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31515)  | [Monolith Env](https://pr-31515-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal PR](https://github.com/codecademy-engineering/portal-app/pull/2224)  | [Portal Env]( https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2224)   |
